### DISCUSSION TITLE
Streaming audio

### DISCUSSION THREAD

**Discussion by bl0rk** [Pycord-Development/pycord#1755] [Q&A] [answered]
Heya people! 
I'm in the process of rewriting my old discord bot with pycord and am now wanting to work on the music part of it. Problem is, while the last version technically worked, it worked by downloading the song using pytube and then playing it, but for some reason the download speed on the server it runs on is abysmal. 
Now i'm wondering if anyone has any other alternatives to play audio/songs with the bot, maybe even by just directly streaming it from a video/music sharing platform.

**Comment by Dorukyum** [comment, marked as answer]
I suggest trying out [pycord.wavelink](https://github.com/Pycord-Development/Pycord.Wavelink) and asking for help in our Discord server next time.
